index.js

```diff
@@ -113,6 +114,10 @@ module.exports = {
   },
 
   init: function (app) {
+    // Determine if the ember version is >= 2.12
+    const checker = new VersionChecker(this)
```

Comment:
Is `this` not available in `treeForAddon()` that this code has to located here rather than in `treeForAddon()`?